recipes/pycontracts/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "PyContracts" %}
+{% set version = "1.8.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/4e/7a/0f79370e4e3a6741396d76d1f76586c2924bed049fb38597799b72a24081/{{ name }}-{{ version }}.tar.gz"
+  sha256: e76adbd832deec28b2045a6094c5bb779a0b2cb1105a23b3efafe723e2c9937a
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - decorator
+    - future
+    - pip
+    - pyparsing
+    - python
+    - six
+  run:
+    - decorator
+    - future
+    - pyparsing
+    - python
+    - six
+
+test:
+  imports:
+    - contracts
+    - contracts.library
+    - contracts.testing
+    - contracts.testing.library
+    - contracts.useful_contracts
+  requires:
+    - nose
+
+about:
+  home: "http://andreacensi.github.com/contracts/"
+  license: "GNU Library or Lesser General Public (LGPL)"
+  license_family: LGPL
+  license_file: LICENSE.txt
+  summary: "PyContracts is a Python package that allows to declare constraints on function parameters and return values. Contracts can be specified using Python3 annotations, in a decorator, or inside a docstring :type: and :rtype: tags. PyContracts supports a basic type system, variables binding, arithmetic constraints, and has several specialized contracts (notably for Numpy arrays), as well as an extension API."
```

Comment:
Can you change this to be the description and make the summary a shorter version